compiler/rustc_metadata/src/creader.rs

```diff
@@ -1195,9 +1195,8 @@ impl<'a, 'tcx> CrateLoader<'a, 'tcx> {
         }
 
         // `compiler_builtins` is not yet in the graph; inject it. Error on resolution failure.
```

Comment:
```suggestion
        // `compiler_builtins` is not yet in the graph; inject it.
```

Reply:
Linker failures are less user friendly that rustc errors. Also it would allow accidentally depending on rustc not emitting any compiler-builtins calls for a given crate and thus linking succeeding with one rustc version and then failing on the next due to an optimization change.

Reply:
Makes sense. I wonder if it would make sense to try the sysroot if it can't resolve, but that doesn't come up very much.

Reply:
This should already try resolving from the sysroot if there is no `--extern compiler_builtins=...`.